### PR TITLE
Cosmetic changes to readport_4001.py

### PR DESCRIPTION
**Pull request by dvukolov**
- Removed a separate data_list counter
- Fixed docstrings

### CODE REVIEW

File: readport_4001.py [hunk 1]
```diff
@@ -1,13 +1,12 @@
 #!/usr/bin/env python3.5
 
+import logging
+import logging.handlers
 import re
 import signal
 import socket
 import time
```
Comment (alexavr) [inline readport_4001.py:8]: some text here

File: readport_4001.py [hunk 2]
```diff
@@ -46,7 +45,7 @@ def interrupt_handler(sig, frame):
 
 
 def configure_logging():
-    """Setup logging to the file and the console
+    """Setup rotated logging to the file and the console
     """
     root = logging.getLogger()
     root.setLevel(LOG_LEVEL)
```
Comment (alexavr) [inline readport_4001.py:51]: test of review

File: readport_4001.py (hunk 1 above)
Comment (dvukolov) [reply readport_4001.py:8]: imports are not alphabetically sorted

**Review by alexavr — COMMENTED** [review]
test

File: readport_4001.py [hunk 3]
```diff
@@ -247,8 +244,8 @@ def parse(data):
         u, v, w, t: extracted float values or FILL_VALUES
 
     Raises:
-        IncompleteDataException: if an unrecoverable incomplete message is received.
-            In thise case, it doesn't make sense to use fill values for u, v, w, t.
+        IncompleteDataException: when an unrecoverable incomplete message is received.
+            In this case, it doesn't make sense to use fill values for u, v, w, t.
     """
     # Test whether a complete message has been received
     pattern = rb"^\x02Q,.*,\x03..\r\n$"
```
Comment (alexavr) [inline readport_4001.py:251]: The second comment

File: readport_4001.py [hunk 4]
```diff
@@ -230,11 +228,10 @@ def process_data(queue):
```
Comment (alexavr) [inline readport_4001.py:228]: The third comment

File: readport_4001.py [hunk 5]
```diff
@@ -230,11 +228,10 @@ def process_data(queue):
...
             data_list = []
-            count = 0
 
 
 def parse(data):
-    """Extract the variabls u, v, w, t from the binary message.
+    """Extract the variables u, v, w, t from the binary message.
 
     Ensures the message is complete, i.e. starts with ".Q" and ends with "\r\n",
     otherwise raises an exception. If unable to extract variables from a complete
```
Comment (alexavr) [inline readport_4001.py:237]: The second review

**Review by alexavr — APPROVED** [review]

**Comment by alexavr** [comment]
Thanx!

**Pull request merged**